src/pages/submit/linked-wearables.tsx

```diff
@@ -154,28 +165,54 @@ const validate = createValidator<LinkedWearablesState>({
         assert(state.motivation.length >= schema.motivation.minLength, 'error.linked_wearables.motivation_too_short') ||
         assert(state.motivation.length <= schema.motivation.maxLength, 'error.linked_wearables.motivation_too_large'),
       method: assert(state.method.length <= schema.method.maxLength, 'error.linked_wearables.method_too_large'),
-      managers:
-        assert(
-          managers.some((option) => option !== ''),
-          'error.linked_wearables.managers_empty'
-        ) ||
-        assert(
-          managers.every((option) => isEthereumAddress(option)),
-          'error.linked_wearables.address_invalid'
-        ),
+      managers: assert(
+        managers.some((option) => option !== ''),
+        'error.linked_wearables.managers_empty'
+      ),
     }
   },
 })
 
+const removeEmptyStrings = (array: string[]) => array.filter((item) => item !== '')
```

Comment:
You could also write this like `array.filter(a =>  a)` since it removes all falsey values